frontend/src/components/CourseModalOverview.js

```diff
@@ -550,8 +553,10 @@ const CourseModalOverview = ({ setFilter, filter, setSeason, listing }) => {
                   target="_blank"
                   rel="noopener noreferrer"
                   href={listing.syllabus_url}
+                  className="d-flex"
                 >
                   {listing['course_code']}
```

Comment:
Maybe we should change the text to "view syllabus" or something?

Also we should move the entire syllabus row up in the modal - probably should be the first item right?